Jenkinsfile

```diff
@@ -56,6 +56,7 @@ pipeline {
       }
       steps {
         deploy("cardid", "test", null, true)
+        deployEcs("cardid", "test", null, true)
```

Comment:
this should be `..., null, true, true` as we want it tag and run tests after deploying to ECS, and we want a tag.

See [https://github.com/alphagov/pay-selfservice/blob/master/Jenkinsfile](https://github.com/alphagov/pay-selfservice/blob/master/Jenkinsfile)